src/platform_impl/linux/drm/window.rs

```diff
@@ -0,0 +1,354 @@
+            event_loop_window_target.plane,
+            find_prop_id(
+                &event_loop_window_target.device,
+                event_loop_window_target.plane,
+                "SRC_X",
+            )
+            .expect("Could not get SRC_X"),
+            property::Value::UnsignedRange(0),
+        );
+        atomic_req.add_property(
+            event_loop_window_target.plane,
+            find_prop_id(
+                &event_loop_window_target.device,
+                event_loop_window_target.plane,
+                "SRC_Y",
+            )
+            .expect("Could not get SRC_Y"),
+            property::Value::UnsignedRange(0),
+        );
+        atomic_req.add_property(
+            event_loop_window_target.plane,
+            find_prop_id(
+                &event_loop_window_target.device,
+                event_loop_window_target.plane,
+                "SRC_W",
+            )
+            .expect("Could not get SRC_W"),
+            property::Value::UnsignedRange((event_loop_window_target.mode.size().0 as u64) << 16),
+        );
+        atomic_req.add_property(
+            event_loop_window_target.plane,
+            find_prop_id(
+                &event_loop_window_target.device,
+                event_loop_window_target.plane,
+                "SRC_H",
+            )
+            .expect("Could not get SRC_H"),
+            property::Value::UnsignedRange((event_loop_window_target.mode.size().1 as u64) << 16),
+        );
+        atomic_req.add_property(
+            event_loop_window_target.plane,
+            find_prop_id(
+                &event_loop_window_target.device,
+                event_loop_window_target.plane,
+                "CRTC_X",
+            )
+            .expect("Could not get CRTC_X"),
+            property::Value::SignedRange(0),
+        );
+        atomic_req.add_property(
+            event_loop_window_target.plane,
+            find_prop_id(
+                &event_loop_window_target.device,
+                event_loop_window_target.plane,
+                "CRTC_Y",
+            )
+            .expect("Could not get CRTC_Y"),
+            property::Value::SignedRange(0),
+        );
+        atomic_req.add_property(
+            event_loop_window_target.plane,
+            find_prop_id(
+                &event_loop_window_target.device,
+                event_loop_window_target.plane,
+                "CRTC_W",
+            )
+            .expect("Could not get CRTC_W"),
+            property::Value::UnsignedRange(event_loop_window_target.mode.size().0 as u64),
+        );
+        atomic_req.add_property(
+            event_loop_window_target.plane,
+            find_prop_id(
+                &event_loop_window_target.device,
+                event_loop_window_target.plane,
+                "CRTC_H",
+            )
+            .expect("Could not get CRTC_H"),
+            property::Value::UnsignedRange(event_loop_window_target.mode.size().1 as u64),
+        );
```

Comment:
All of those `atomic_req` stuff is looking the same, consider adding a function or macro for them. Will likely work in some other places.

Reply:
Fixed